QuasarHelios2015/src/org/team1540/quasarhelios/Clamp.java

```diff
@@ -45,6 +47,11 @@
     public static BooleanInput atBottom;
     private static final BooleanStatus needsAutoCalibration = new BooleanStatus(useEncoder.get()); // yes, this only sets the default value at startup.
 
+    public static final BooleanStatus enabled = new BooleanStatus(true);
+
+    private static BooleanInput modeOutput = BooleanMixing.orBooleans(enabled.asInvertedInput(), mode);
+    private static FloatInput speedOutput = FloatMixing.createDispatch(Mixing.select((BooleanInputPoll) enabled, FloatMixing.always(0), speed), Igneous.globalPeriodic);
```

Comment:
What name do you recommend? I don't want to call it ``Input``, because although it is an input, we are not actually inputing a value for it so it would be sort of misleading. Would ``actualSpeed`` work?